From 0ec8d02b7d0404c6408d104910930af11c4be29f Mon Sep 17 00:00:00 2001
From: chunguang yang <chunguang.yang@windriver.com>
Date: Thu, 25 Jun 2015 11:16:24 +0800
Subject: [PATCH] R8168: Add vlan support

R8168 disables vlan by default and as a result, when the
interface is configured for vlan, scp or iperf tranfers
on the vlan subinterface will drop connection and fail.

If you need vlan support, and not scp or iperf, enable
this option.

Signed-off-by: chunguang yang <chunguang.yang@windriver.com>
---
 drivers/staging/r8168/Kconfig |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/r8168/Kconfig b/drivers/staging/r8168/Kconfig
index b1ff974..34aba00 100644
--- a/drivers/staging/r8168/Kconfig
+++ b/drivers/staging/r8168/Kconfig
@@ -29,5 +29,12 @@ config R8168
 	  To compile this driver as a module, choose M here: the module
 	  will be called r8168.  This is recommended.
 
+config R8168_VLAN
+        bool "Realtek 8168 vlan support"
+        depends on R8168
+        default y
+        ---help---
+          Say Y here if you want to Realtek 8168 support vlan.
+
 endif # NET_VENDOR_REALTEK
 
-- 
1.7.5.4

